padl/dumptools/var2mod.py

```diff
@@ -611,11 +619,11 @@ def _check_and_make_increment(var: ScopedName, scope: Scope, scoped_name: Scoped
     :return:
         ScopedName after increment
     """
+    new_scoped_name = ScopedName(scoped_name.name, scope, var.n + scoped_name.n + 1)
     try:
-        new_scoped_name = ScopedName(var.name, scope, var.n + scoped_name.n + 1)
         find_in_scope(new_scoped_name)
     except NameNotFound:
-        new_scoped_name = ScopedName(var.name, scope, var.n + scoped_name.n)
+        new_scoped_name = ScopedName(scoped_name.name, scope, var.n + scoped_name.n)
```

Comment:
Why are we using `scoped_name.name` here instead of `var.name`?

Reply:
I updated this using `var.name`.  `var.name` makes more sense.